meetings/agenda.md

```diff
@@ -6,3 +6,19 @@
 Please file pull requests to add, or discuss items to add, to the agenda.
 
 ## Items to Discuss
+
+### PHP Download & Installation Instructions
+
+Do we want to be mentioned in the [PHP downloads](https://www.php.net/downloads.php) page ?
+In this case, we need to create a setup page in documentation, about how to install Mezzio.
```

Comment:
I doubt that this will be enough, but we should talk about it.

Btw. there is another PR which already includes Laminas: https://github.com/php/web-php/pull/1172/files#diff-7f892a85a689a07c4bdf0ec1c3ee014035f5f1a25b513d055eb6fac935ca5448R189-R193 😃